__tests__/postService.test.js

```diff
@@ -1,5 +1,5 @@
-const { createPost } = require("./src/services/postService");
-const { sendPost } = require("./src/repository/postDAO");
+const { createPost } = require("../src/services/postService");
+const { sendPost } = require("../src/repository/postDAO");
 
 describe('createPost test', () => {
```

Comment:
`sendPost` is not actually mocked here, resulting in it actually sending data to dynamoDB. DAO should be mocked in a `beforeAll` block, check `userService.test.js` for an example.